ocaml/idl/datamodel.ml

```diff
@@ -2791,6 +2791,8 @@ module Sr_stat = struct
       , [
           ("healthy", "Storage is fully available")
         ; ("recovering", "Storage is busy recovering, e.g. rebuilding mirrors.")
+        ; ("unreachable", "Storage is unreachable")
+        ; ("unavailable", "Storage is unavailable")
```

Comment:
Please update the `~lifecycle` arg of the `health` field definition below, which this enum is used to indicate the change. E.g.:

    ~lifecycle:[(Prototyped, rel_kolkata, ""); (Published, rel_lima, ""); Extended, "24.16.0", "Enum extended with 'unreachable' and 'unavailable' values"]